scripts/teigaku_genzei/json_qa.py

```
@@ -0,0 +1,86 @@
+def extract_qa_pairs(data):
+    qa_pairs = []
+    text_list = data["texts"]
+    title_list = filter(lambda x: json_util.is_title(x[1]["text"]), enumerate(text_list))
+    qhead_list = filter(lambda x: json_util.is_qhead(x[1]["text"]), enumerate(text_list))
+    ahead_list = filter(lambda x: json_util.is_ahead(x[1]["text"]), enumerate(text_list))
+    #－
+    #ー
+    title_pos_list = [ti for (ti, title) in title_list]
+    qhead_pos_list = [qi for (qi, qhead) in qhead_list]
+    ahead_pos_list = [ai for (ai, qhead) in ahead_list]
+    atail_pos_list = [i for i in title_pos_list][1:] + [len(text_list)]
```

Pylint (4.0.0)

[refactor] Unnecessary use of a comprehension, use list(title_pos_list) instead.

(R1721)